### PR TITLE
labels support docs update

### PR DESCRIPTION
**Pull request by Enkumicahel**
labels support docs update

### CODE REVIEW

File: content/yaml/yaml-getting-started.md
Comment (mikhail-tokarev) [inline content/yaml/yaml-getting-started.md:135]: we use 2 spaces everywhere and let's provide better name to make it self-descriptive

File: content/yaml/yaml-getting-started.md
Comment (mikhail-tokarev) [inline content/yaml/yaml-getting-started.md:471]: `Labels also supports` - grammar?

File: content/yaml/yaml-getting-started.md
Comment (mikhail-tokarev) [inline content/yaml/yaml-getting-started.md:476]: 2 spaces

**Review by mikhail-tokarev — CHANGES_REQUESTED** [review]

File: content/yaml/yaml-getting-started.md
Comment (mikhail-tokarev) [inline content/yaml/yaml-getting-started.md:478]: `labels` should be on the same level as workflow name

**Review by mikhail-tokarev — CHANGES_REQUESTED** [review]

File: content/yaml/yaml-getting-started.md
Comment (helinanever) [inline content/yaml/yaml-getting-started.md:472]: What's missing for me here is information about why is this useful and what can I do with them after I define them

File: content/yaml/yaml-getting-started.md
Comment (helinanever) [inline content/yaml/yaml-getting-started.md:134]: I'm a bit confused. Are labels part of `publishing` or are they defined at the workflow level (as in the example below) or are both supported?

Comment (Enkumicahel) [reply content/yaml/yaml-getting-started.md:134]: Updated [PR](https://github.com/codemagic-ci-cd/codemagic-docs/pull/913/commits/c90e4253b75e66cd931ef11d3a0f16c2df514386)

Comment (helinanever) [reply content/yaml/yaml-getting-started.md:134]: `labels` are still under `publishing` and not directly under `workflow` here.

File: content/yaml/yaml-getting-started.md
Comment (helinanever) [inline content/yaml/yaml-getting-started.md:445]: Nice, but I think it can be improved a little. We will first support displaying the labels and then soon after also filtering builds based on them. We'll update this part in docs when needed.
```suggestion
You may use `codemagic.yaml` to define labels for your apps. Labels serve as additional information about the workflow you are building and are helpful when you have multiple versions of a workflow, e.g. when you build white label apps. The labels are visible on the `/builds` and `/app/<app-id>/build/<build-id>` pages. As shown in the snippet below, labels also support environment variables.
```

**Review by helinanever — CHANGES_REQUESTED** [review]

**Review by helinanever — APPROVED** [review]
Looks good to me. We need to wait for frontend changes before we can merge the PR though.

**Review by mikhail-tokarev — APPROVED** [review]
LGTM

**Pull request merged**
